### PR TITLE
tools: Add table script utility similar to vptscript in vpx-js

### PR DESCRIPTION
**Pull request by jsm174**
The PR adds a utility for extracting table scripts. I needed something like `vptscript` in `vpx-js`, but I also needed it to convert all the different styles of carriage returns found in the table scripts on non Windows systems.

### CODE REVIEW

**Comment by jsm174** [comment]
Forgot to add CI.. Will add.

File: VisualPinball.TableScript/Program.cs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+					outputDir = args[1];
+					if (!Directory.Exists(outputDir)) {
+						throw new ArgumentException($"Invalid output folder \"{outputDir}\".");
+					}
+				}
+
+				foreach (var inputFile in inputFiles) {
+					Console.WriteLine($"Processing \"{inputFile}\"...");
+
+					var inputTable = TableLoader.Load(inputFile);
```
Comment (freezy) [inline VisualPinball.TableScript/Program.cs]: You can add `false` here as second parameter so it doesn't load the game items, which should be a bit faster. You could even add another parameter to skip textures and sounds for extreme speed.

**Review by freezy — APPROVED** [review]
Awesome, thanks a lot!

File: VisualPinball.TableScript/Program.cs (hunk 1 above)
Comment (jsm174) [reply VisualPinball.TableScript/Program.cs]: awesome. Added `false`!

**Pull request merged**